types/shelljs/index.d.ts

```diff
@@ -1236,7 +1236,7 @@ export interface ShellConfig {
 	fatal: boolean;
 
 	/**
-	 * Will print each executed command to the screen. Default is true.
+	 * Will print each executed command to the screen. Default is false.
```

Comment:
It might be a good idea to use the `@default` TSDoc tag:
```suggestion
	 * Will print each executed command to the screen.
	 *
	 * @default false
```